src/components/CalendarContainer.tsx

```diff
@@ -109,6 +109,11 @@ export interface CalendarContainerProps<T extends ICalendarEventBase> {
   disableMonthEventCellPress?: boolean
   showVerticalScrollIndicator?: boolean
   itemSeparatorComponent?: React.ComponentType<any> | null | undefined
+  /**
+   * Callback when the user swipes horizontally.
+   * @param date The date where the user swiped to.
+   */
+  onSwipe?: (date: Date) => void
```

Comment:
Should it be `onSwipeEnd`
or do you plan to call it with some throttle behavior each X ms?

Reply:
onSwipeEnd is more accurate actually, I changed it